developing instructions: mention that issues are not written perfectly (#532)
Akuli/porcupine: README.md

If you want to do something to Porcupine, that's awesome!
I have tried to make contributing easy:

> - If you don't understand what I meant in an issue, please ask for clarification.

Author: ```suggestion
- If you don't understand what I meant in an issue, please ask me to clarify it.
```
